pkg/stanza/operator/helper/multiline.go

```diff
@@ -30,6 +32,7 @@ func NewMultilineConfig() MultilineConfig {
 type MultilineConfig struct {
 	LineStartPattern string `mapstructure:"line_start_pattern"`
 	LineEndPattern   string `mapstructure:"line_end_pattern"`
+	OctetCounting    bool   `mapstructure:"enable_octet_counting"`
```

Comment:
Only problem I have with this solution is that octet counting is specific to syslog, whereas multiline config is not. Can we move this parameter and splitfunc to the syslog input?

Reply:
i may be missing something in the flow. how can you have the splitfunc in the syslog input and pass it to the tcp input to split https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/operator/input/tcp/tcp.go#L246?

Reply:
The syslog input builds the splitfunc here, so we could wrap the function with another that splits syslog entries.